chrome_extension/background/init.js

```diff
@@ -139,13 +139,11 @@ startMooltipass = function() {
 
 			// Intercept posts
 			if (details.method == "POST") {
-				// Deal both with RAW DATA and FORM DATA
-				if (details && details.type === "xmlhttprequest") { // Raw data (multipart posts, etc)
-					if ( details.requestBody && details.requestBody.raw && details.requestBody.raw[0]) {
-						var buffer = details.requestBody.raw[0].bytes;
-						var parsed = arrayBufferToData.toJSON(buffer);
-						if ( details.tabId ) chrome.tabs.sendMessage( details.tabId, {action: 'post_detected', post_data: parsed });	
-					} 
+				// Deal with RAW DATA
```

Comment:
The main problem with this code is that when request has _xmlhttprequest_ type, it doesn't mean that it's raw data. Previously we lose such requests.